ChartboostMediationAdapterBidMachine.podspec

```diff
@@ -15,7 +15,7 @@ Pod::Spec.new do |spec|
 
   # Minimum supported versions
   spec.swift_version         = '5.1'
-  spec.ios.deployment_target = '12.0'
+  spec.ios.deployment_target = '13.0'
```

Comment:
Do we need to upgrade? BidMachine 3.0.0 supports iOS 12+ and this adapter is not on 5.x yet.
https://github.com/CocoaPods/Specs/blob/master/Specs/4/0/5/BidMachine/3.0.0/BidMachine.podspec.json

Reply:
+1
It's not uncommon for ad SDKs to have a minimum supported version while keeping an older one as the podspec deployment target. This way they can still be integrated in apps that have to support old OS versions, although they don't serve ads for them.

Reply:
The version should be the same as the partner podspec.

Reply:
Lies! Lies in the podspec!
![Screenshot 2024-07-26 at 12 56 23 PM](https://github.com/user-attachments/assets/5dfebefe-622d-4aeb-8371-d71dd2dfbd96)

Reply:
That's amazing 😂